Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1339,6 +1367,11 @@ extension BrowserViewController : Transitionable {
             }
         }
         self.homePanelController?.view.hidden = false
+        if options.toView === self {
+            startTrackingAccessibilityStatus()
+        } else {
+            stopTrackingAccessibilityStatus()
+        }
```

Comment:
Btw. I know I have been behind with testing with all my commits so far, as accessibility is not yet finished so at this point the user will benefit more from more things working in the first released version, and that I will be able to get back to add some tests later.